src/utils/validators_enhanced.py

```
@@ -0,0 +1,453 @@
+    """Backward compatible nickname validation function."""
+    try:
+        return bool(validate_nickname_enhanced(nickname))
+    except Exception:
+        return False
+
+
+def validate_password(password: str) -> bool:
+    """Backward compatible password validation function."""
+    try:
+        return bool(validate_password_enhanced(password))
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)